server/src/main/scala/za/co/absa/atum/server/api/controller/PartitioningControllerImpl.scala

```diff
@@ -49,12 +50,6 @@ class PartitioningControllerImpl(partitioningService: PartitioningService)
     atumContextDTOEffect
   }
 
-  override def createPartitioningIfNotExistsV2(
```

Comment:
If you remove this one, won't it remove the v1 endpoint (make it inoperable?

Reply:
According to @salamonpavel 
_There is createPartitioningIfNotExistsV1 method for the V1 endpoint._